src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Test/Index.cshtml

```diff
@@ -41,7 +46,7 @@
                             </div>
                         }
 
-                        if (Model.Discount != null && Model.IsAdvent)
+                        if (Model.Discount != null && isAdvent)
```

Comment:
it's ok, but we can change the order - first check isAdvent and then check model